internal/operators/ocs/ocs_operator.go

```diff
@@ -142,15 +142,15 @@ func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluste
 		// for each disk ocs requires 2 CPUs and 5 GiB RAM
 		if host.Role == models.HostRoleMaster || host.Role == models.HostRoleAutoAssign {
 			return &models.ClusterHostRequirementsDetails{
-				CPUCores:   CPUCompactMode + reqDisks*o.config.OCSRequiredDiskCPUCount,
-				RAMMib:     conversions.GibToMib(MemoryGiBCompactMode + reqDisks*o.config.OCSRequiredDiskRAMGiB),
+				CPUCores:   CPUCompactMode + reqDisks*o.config.OCSPerDiskCPUCount,
```

Comment:
Please use () here as well. Sorry for missing this in the last review.

Reply:
Done